get/releases.go

```diff
@@ -0,0 +1,94 @@
+package get
+
+import (
+	"context"
+	"os"
+	"sort"
+	"strconv"
+	"text/tabwriter"
+
+	apps "github.com/ninech/apis/apps/v1alpha1"
+	"github.com/ninech/nctl/api"
+	"github.com/ninech/nctl/api/util"
+)
+
+type releasesCmd struct {
+	Name string `arg:"" help:"Name of the Application for which to get the Releases. If omitted all Applications will be listed." default:""`
```

Comment:
Can we also add an optional flag to get releases by application name? See `Build` for an example.